### PR TITLE
New JavaScript SDK docs

### PR DESCRIPTION
**Pull request by jplace**
I am playing with moving the SDK documentation to the transposit-js-sdk Git repository.

In this diff:
- I deleted the JS SDK reference page from our documentation. It has been moved to the JS SDK repository.
- I rewrote the JS SDK building page to focus on using the SDK from a non-hosted app. I link to the SDK repository for more documentation.

This all only makes sense in the context of the documentation I've added to the JavaScript SDK. It is live now at https://github.com/transposit/transposit-js-sdk . The README is meant to be a quick start. The FAQ links to documentation I plan to build out with details about all the power the SDK has to offer.

What do you think of this?

### CODE REVIEW

File: docs/get-started/terms.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Terms & basics
 
 ## Applications
 
-An application is the representation of a particular project; a container functionality you want to build. Applications contain operations, data connections, keys to access data connections, and configurations for authentication and deployment.
+An application is the representation of a particular project; a container for functionality you want to build. Applications contain operations, data connections, keys to access data connections, and configurations for authentication and deployment.
```
Comment (jplace) [inline docs/get-started/terms.md]: This is a random typo I noticed

File: docs/references/js-sdk.md [hunk 2]
```diff
@@ -1,182 +0,0 @@
----
```
Comment (jplace) [inline docs/references/js-sdk.md]: I need to update all other places in the docs linking to this page. I have broken some links here.

File: docs/building/js-sdk.md [hunk 3]
```diff
@@ -3,120 +3,50 @@ id: js-sdk
...
+5. Save the configuration
+6. Optionally, choose to **Restrict to specific whitelisted domains and users**
 
-Add via a script tag:
+### Disable user sign-in
 
-```markup
-<script src="https://unpkg.com/transposit@0.7.2/dist/bundle.prod.js"></script>
-```
+1. Navigate to **Authentication** > **User Sign-in**
```
Comment (jplace) [inline docs/building/js-sdk.md]: These carrots should be in format like:
```
**Authentication &gt; User Sign-in**
```

File: docs/building/js-sdk.md [hunk 4]
```diff
@@ -3,120 +3,50 @@ id: js-sdk
...
-Transposit's JavaScript SDK makes it simple to deal with sign-in and authentication, and to run operations in your application.
+Use the Transposit JavaScript SDK to call operations from a web app.
 
-Install with npm:
+## Configure user sign-in
 
-```text
-npm install transposit
-```
+Your web app can be available to the public or protected by Google sign-in. Web apps protected by sign-in can allow access only to certain email addresses or email domains.
```
Comment (jplace) [inline docs/building/js-sdk.md]: This copy should more closely resemble what's written on the hosted app page.

File: docs/building/js-sdk.md (hunk 3 above)
Comment (jpwain) [reply docs/building/js-sdk.md]: 👍 Also, where should we document our documentation-writing standards? that navigation paths should be written this way seems like something to add to some code styleguide type document somewhere.

File: docs/building/js-sdk.md [hunk 5]
```diff
@@ -3,120 +3,50 @@ id: js-sdk
 title: JavaScript SDK
 ---
 
-Transposit's JavaScript SDK makes it simple to deal with sign-in and authentication, and to run operations in your application.
+Use the Transposit JavaScript SDK to call operations from a web app.
```
Comment (jpwain) [inline docs/building/js-sdk.md]: @jplace I expected this instance of `Transposit JavaScript SDK` to be linked to the other repo as well; I didn't find the bottom link until I was already confused.

Comment (jplace) [reply docs/building/js-sdk.md]: fixed

File: docs/building/js-sdk.md (hunk 4 above)
Comment (jplace) [reply docs/building/js-sdk.md]: fixed

File: docs/building/js-sdk.md (hunk 3 above)
Comment (jplace) [reply docs/building/js-sdk.md]: fixed

**Pull request merged**